sql/core/src/test/scala/org/apache/spark/sql/execution/joins/HashedRelationSuite.scala

```diff
@@ -341,6 +341,45 @@ class HashedRelationSuite extends SparkFunSuite with SharedSQLContext {
     assert(java.util.Arrays.equals(os.toByteArray, os2.toByteArray))
   }
 
+  test("SPARK-31511: Make BytesToBytesMap iterators thread-safe") {
```

Comment:
can you open a new PR to add this test for master/3.0?

Reply:
ok, I submit a pull request to add test.
https://github.com/apache/spark/pull/29669